src-tauri/src/northstar/profile.rs

```diff
@@ -0,0 +1,73 @@
+use crate::GameInstall;
+
+// These folders are part of Titanfall 2 and
+// should NEVER be used as a Profile
+const SKIP_PATHS: [&str; 8] = [
+    "___flightcore-temp",
+    "__overlay",
+    "bin",
+    "Core",
+    "r2",
+    "vpk",
+    "platform",
+    "Support",
+];
+
+// A profile may have one of these to be detected
+const MAY_CONTAIN: [&str; 10] = [
+    "mods/",
+    "plugins/",
+    "packages/",
+    "logs/",
+    "runtime/",
+    "save_data/",
+    "Northstar.dll",
+    "enabledmods.json",
+    "placeholder.playerdata.pdata",
+    "LEGAL.txt",
+];
+
+#[tauri::command]
+pub fn fetch_profiles(game_install: GameInstall) -> Result<Vec<String>, String> {
```

Comment:
I'd add some [short doc](https://doc.rust-lang.org/reference/comments.html#doc-comments) comment to the function.